_shared/project/setup.cfg

```diff
@@ -20,25 +20,26 @@ packages = find:
 python_requires = >={{ python_versions()|oldest|pyformat(PyFormats.MAJOR_DOT_MINOR_FMT) }}
 install_requires =
     importlib_metadata;python_version<"3.8."
-    {%- if include_exists("setuptools/install_requires") %}
-      {{- include("setuptools/install_requires", indent=4) }}
-    {%- endif %}
+{% if include_exists("setuptools/install_requires") %}
+{{ include("setuptools/install_requires", indent=4) -}}
+{% endif %}
 
 [options.packages.find]
 where = src
 
 [options.entry_points]
-{%- if include_exists("setuptools/console_scripts") %}
+{% if include_exists("setuptools/console_scripts") %}
 console_scripts =
-    {{- include("setuptools/console_scripts", indent=4) }}
-{%- elif cookiecutter.get("console_script") == "yes" %}
+{{ include("setuptools/console_scripts", indent=4) -}}
+{% elif cookiecutter.get("console_script") == "yes" %}
```

Comment:
Similar question about indentation here etc.

Reply:
It's not possible to indent this because the body of the `{% if %}` contains more than one line. If you simply indent the `{{ include(...) -}}` line that will indent the first line from the `setuptools/console_scripts` include file. If you indent the `include` line with a `{{-` then it'll run that line into the `console_scripts =` line above.